ee-9/feature-pack/pom.xml

```diff
@@ -507,6 +507,10 @@
                                 <exclude>io.opentelemetry:opentelemetry-sdk-common\z</exclude>
                                 <exclude>io.opentelemetry:opentelemetry-sdk-trace\z</exclude>
                                 <exclude>io.opentelemetry:opentelemetry-semconv\z</exclude>
+                                <exclude>io.opentracing.contrib:opentracing-jaxrs2\z</exclude>
+                                <exclude>io.opentracing.contrib:opentracing-interceptors\z</exclude>
+                                <exclude>io.opentracing.contrib:opentracing-tracerresolver\z</exclude>
+                                <exclude>io.opentracing.contrib:opentracing-web-servlet-filter\z</exclude>
```

Comment:
In this part we are grouping items based on the preceding comment that explains the exclude. And then alphabetically. Are these all excluded because of this?

```
<!-- Don't transform libraries that use the javax.annotation package but
        not the Jakarta Annotations APIs that share it with other APIs.
        Generally (perhaps always), this is from use of JSR 305 annotations -->
```

My comments are all about code organization. :)

Reply:
TBH, I'm not sure. :) They do need to be excluded (the current state of this PR has them commented out) and it shows they're being transformed. I put them here because that's where I put the OpenTelemtry artifacts (which may also be in the wrong place).

Reply:
Lines removed.